de/dynamic_data_in_templates/README.md

```diff
@@ -27,23 +26,23 @@ from django.shortcuts import render
 from .models import Post
 ```
 
-Der Punkt vor `models` bedeutet *current directory*, also das aktuelle Verzeichnis, oder *current application*, die aktuelle Anwendung. Die `views.py` und `models.py` liegen im selben Verzeichnis. Dadurch können wir `.` und den Namen der Datei (ohne `.py`) benutzen. Dann ergänzen wir für den Import den Namen des Models (`Post`).
+Der Punkt vor `models` bedeutet *aktuelle Verzeichnis* oder *aktuelle Anwendung*. Die Dateien `views.py` und `models.py` liegen im selben Verzeichnis. Deswegen können wir `.` und den Namen der Datei (ohne `.py`) benutzen. Dann ergänzen wir für den Import den Namen des Models (`Post`).
```

Comment:
Changed to "aktuelles Verzeichnis" on crowdin.

Reply:
oops :flushed: